src/main/java/net/sf/mcf2pdf/mcfelements/util/ImageUtil.java

```diff
@@ -116,6 +117,11 @@ public static BufferedImage readImage(File imageFile) throws IOException {
 	}
 
 	private static int getImageRotation(File imageFile) throws IOException {
+		// ToDo: determine rotation for png's, method below leads to crash
+		if (imageFile.getName().toLowerCase(Locale.US).endsWith(".png")) {
+			return 0;
+		}
+		

```

Comment:
Sorry, can't accept removal of functionality without proper replacement; please create issue describing the crash (including stacktrace). Best would be if crash-causing PNG could be attached. Thanks!